### PR TITLE
Revert "fix(ec): use only results from `report-json` step"

### PR DESCRIPTION
**Pull request by sahil143**
Reverts openshift/hac-dev#971

https://issues.redhat.com/browse/KFLUXBUGS-1577

### CODE REVIEW

**Comment by Katka92** [comment]
[test]

File: src/components/EnterpriseContractView/useEnterpriseContractResultFromLogs.tsx [hunk 1]
```diff
@@ -15,39 +18,81 @@ export const useEnterpriseContractResultFromLogs = (
...
   const [ecLoaded, setEcLoaded] = React.useState<boolean>(false);
+  const ecResultOpts = React.useMemo(() => {
+    const podName = loaded && !error ? taskRun[0].status.podName : null;
+    return podName
+      ? {
+          ns: namespace,
+          name: podName,
+          path: 'log',
+          queryParams: {
+            container: 'step-report',
```
Comment (jrichter1) [inline src/components/EnterpriseContractView/useEnterpriseContractResultFromLogs.tsx:32]: given the point of #971 was to stop using `report` step, this should probably pull logs from `report-json` step for future use

Comment (sahil143) [reply src/components/EnterpriseContractView/useEnterpriseContractResultFromLogs.tsx:32]: Yes, I will make that change

**Comment by sahil143** [comment]
[test]

**Pull request merged**